### PR TITLE
Makefile: split tools/library install targets for cross and native

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

**Review by brobecke — COMMENTED** [review]
I am wondering if adding "install-clean" to the dependencies of all 4 new install-*-* rules is what we want here. I think it might be causing problems uninstalling stuff you don't want uninstalled, if you're not careful.

Shouldn't we instead keep this as the first dependency of install-native and install-cross?

What do you think, @TurboGit ?

**Comment by TurboGit** [comment]
Agreed, to me `install-clean` should the first dependency on `install-cross` and `install-native` targets.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Fix install-clean. TIA.

**Comment by pmderodat** [comment]
Done, thank you for the suggestion!

File: Makefile [hunk 1]
```diff
@@ -258,29 +258,37 @@ endif
...
 		$(GPR_OTHER) --build-name=$(OTHER_LIBRARY_TYPE) aws.gpr
 endif
 
-install-cross: install-clean
+install-tools-native:
+	$(GPRINSTALL) $(GPROPTS) $(GPRINST_OPTS) $(GPR_STATIC) --mode=usage \
+		--build-name=$(DEFAULT_LIBRARY_TYPE) \
+		--install-name=aws tools/tools.gpr
+
+install-native: install-lib-native install-tools-native
```
Comment (brobecke) [inline Makefile:274]: Did you mean to add an `install-clean` here?

**Review by brobecke — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (pmderodat) [reply Makefile:274]: Ah yes, sorry… Should be fixed now, thanks!

**Review by brobecke — APPROVED** [review]
Looks good to me, Pierre-Marie. Thank you.

**Comment by brobecke** [comment]
@TurboGit IIUC the GitHub log correctly, the change is still blocked pending your approval (which is not necessarily a bad thing in this case). Would you mind reviewing too?

Thank you!

**Review by TurboGit — APPROVED** [review]
Looks good now, thanks !

**Comment by pmderodat** [comment]
Great, thank you both for the review!

**Pull request merged**